scripts/openroad/gui.tcl

```diff
@@ -12,4 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 source $::env(SCRIPTS_DIR)/openroad/common/io.tcl
-read
\ No newline at end of file
+read
+if { [info exists ::env(CURRENT_SPEF)] } {
+    puts "Reading spef $::env(CURRENT_SPEF)"
+    read_spefs
+} else {
+    puts "WARNING no spef found for the current run. Please either run estimate_parasitics -placement or estimate_parasitics -global_routing"
```

Comment:
Not really in-line with how we print warnings and such in OpenLane...

Reply:
I don't want to source utils.tcl so I guess I will duplicate puts_warn